classes/core/PKPContainer.inc.php

```diff
@@ -184,7 +196,19 @@ protected function loadConfiguration()
             'auth_mode' => null,
         ];
 
-        $this->instance('config', new Repository($items)); // create instance and bind to use globally
+        // Cache configuration
+        $items['cache'] = [
```

Comment:
Later on, this should be configurable, to allow the user to use a distributed caching solution.